components/List/src/TypographyThemer/MDCListTypographyThemer.h

```diff
@@ -24,10 +24,9 @@
  details on replacement APIs.
  Learn more at docs/theming.md#migration-guide-themers-to-theming-extensions
  */
-@interface MDCListTypographyThemer : NSObject
-@end
-
-@interface MDCListTypographyThemer (ToBeDeprecated)
+__deprecated_msg("Please use MDCSelfSizingStereoCell+MaterialTheming instead. (Note: Typography "
+                 "theming is no longer available as an independent API.)")
+    @interface MDCListTypographyThemer : NSObject
```

Comment:
Should we do something like [this](https://github.com/material-components/material-components-ios/blob/develop/components/NavigationDrawer/src/ColorThemer/MDCBottomDrawerColorThemer.m#L19-L22) in the implementation file to stifle warnings?

Reply:
Done.